src/multivariate/vonmisesfisher.jl

```diff
@@ -65,7 +65,7 @@ function _vmflck(p, κ)
     T = typeof(κ)
     hp = T(p/2)
     q = hp - 1
-    q * log(κ) - hp * log2π - log(besseli(q, κ))
+    q * log(κ) - hp * log2π - log(besselix(q, κ)) + κ
```

Comment:
I think this should be
```suggestion
    q * log(κ) - hp * log2π - log(besselix(q, κ)) - κ
```
?

Reply:
Ahhhh, don't just remove parentheses I guess without thinking :)